snuba/cli/migrations.py

```diff
@@ -85,7 +85,6 @@ def migrate(
     Blocking migrations will not be run unless --force is passed.
     """
     setup_logging(log_level)
-    check_clickhouse_connections()
```

Comment:
this is not strictly necessary. We should update this to only check clusters we will be migrating. Although it doesn't really guarantee that we won't lose connection while applying other migrations.